src/Tribe/Admin/Notices.php

```diff
@@ -137,26 +211,29 @@ public function maybe_display_plus_commerce_notice() {
 		$plus = sprintf(
 			'<a target="_blank" rel="noopener nofollow" href="%s">%s</a>',
 			esc_attr( $plus_link ),
-			esc_html( 'Event Tickets Plus', 'event-tickets' )
+			esc_html__( 'Event Tickets Plus', 'event-tickets' )
 		);
 
-		$plus_commerce_providers = array(
-			esc_html( 'WooCommerce', 'event-tickets' )            => 'woocommerce/woocommerce.php',
-			esc_html( 'Easy Digital Downloads', 'event-tickets' ) => 'easy-digital-downloads/easy-digital-downloads.php',
-		);
+		$plus_commerce_providers = [
+			'woocommerce/woocommerce.php'                       => __( 'WooCommerce', 'event-tickets' ),
```

Comment:
:warning: Array double arrow not aligned correctly; expected 1 space(s) between "'woocommerce/woocommerce.php'" and double arrow, but found 23.

<sub>hash: d2b1653e479fc81c3157ee68c7012cb6</sub>